src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -130,26 +130,20 @@ def _handle_global_configuration():
         global_config.set('logging', 'enable_log_file', 'yes' if enable_file_logging else 'no')
         set_global_config(global_config)
 
-def handle_configure(section=None, name=None, value=None, default_resource_group=None):
+def handle_configure(setting=None, value=None):
     '''
     configure common settings
-    :param str section: configuration section
-    :param str name:    configuration variable name
+    :param str setting: configuration variable, e.g. resource-group, appservice/webapp, compute/vm
```

Comment:
Since we are here, should we support to set multiple values at the same time? Say use a space separated name=value pairs
`--settings resource_group=rg1 appservice/web=myweb compute/vm myvm1`